### ISSUE TITLE
Style Mentors, FAQ and Apply Now similarly to Homepage

### ISSUE THREAD

**Issue by wei2912** [AdvisorySG/mentorship-page#674]
As usual, this should be branched from `multi-page`.

Blocked by merging of https://github.com/AdvisorySG/mentorship-page/pull/667.

**Comment by wei2912** [comment]
Closing since this was already done.
